modules/bootstrapped/src/generated/smithy4s/example/AddMenuItemRequest.scala

```diff
@@ -15,8 +15,8 @@ object AddMenuItemRequest extends ShapeTag.Companion[AddMenuItemRequest] {
   val hints: Hints = Hints.empty
 
   object Optics {
-    val restaurant = Lens[AddMenuItemRequest, String](_.restaurant)(n => a => a.copy(restaurant = n))
-    val menuItem = Lens[AddMenuItemRequest, MenuItem](_.menuItem)(n => a => a.copy(menuItem = n))
+    val restaurantLens = Lens[AddMenuItemRequest, String](_.restaurant)(n => a => a.copy(restaurant = n))
```

Comment:
is it necessary to have the `Lens` suffix ?

Reply:
Scala 2.12 was complaining that it couldn't tell if the name inside of the .copy(myName = ???) was the name of the parameter of the case class or the name of the val.. not sure if there is another way around this or not, but that's why I added the suffix to disambiguate

Reply:
strange, this seems to compile for me:

```scala
//> using scala "2.12.18"
trait Lens[A, B]

object Lens {
  def apply[A, B](f: A => B)(g: (A, B) => A): Lens[A, B] = ???
}

case class Hello(name: String)

object Hello {

  val name: Lens[Hello, String] = Lens[Hello, String](_.name)((c, n) => c.copy(name = n))
}
```

Reply:
Ahh the issue is that 2.12 needs the type annotation. The following fails:

```scala
//> using scala 2.12

trait Lens[A, B]

object Lens {
  def apply[A, B](f: A => B)(g: B => A => A): Lens[A, B] = ???
}

case class Hello(name: String)

object Hello {
  val name  = Lens[Hello, String](_.name)(n => c => c.copy(name = n))
}
```